readthedocs/core/utils/__init__.py

```diff
@@ -150,6 +151,11 @@ def prepare_build(
         # Send Webhook notification for build triggered.
         send_notifications.delay(version.pk, build_pk=build.pk, email=False)
 
+    options['priority'] = CELERY_HIGH
+    if version.type == EXTERNAL:
```

Comment:
Want we check for being a translation here before checking if external and assigning `CELERY_MEDIUM`?